### ISSUE TITLE
[bug]: Cannot invite to Workspace: "You are not a valid viewer. Contact your team owner."

### ISSUE THREAD

**Issue by wicol** [hoppscotch/hoppscotch#4428]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I and my colleague both logged in to the Hoppscotch desktop app via our (work) google accounts.
When we try to invite each other to a workspace we get a success message saying an email has been sent.
None of us are receiving any email, and the invitation link doesn't work, giving us the same error message: "You are not a valid viewer. Contact your team owner.".
Invitations are visible as pending.

Is there something we're missing here or is this a bug?

### Steps to reproduce

See description above.

### Environment

Production

### Version

Cloud

**Comment by liyasthomas** [comment]
Hi @Wicol,

To assist you further, could you please confirm the following:

- Have you checked your Spam folder?
- Are there any email restrictions on certain domains or addresses? If so, please whitelist hoppscotch.io.
- Are you logged in with the correct email while verifying the invite link?
- Please note, the invitation should be sent from the workspace owner.

**Comment by wicol** [comment]
I have checked my spam folder, nothing there. We're using google workspace for email. No special filtering in place.

But emails aside, the invite links copied from the UI don't work either. We very much confirmed we are using the correct email addresses as well.
Don't know if it helps but I did try loading the invite link meant for my colleague, which gave me a different error message saying my email doesn't match the one invited - so that checks out at least.
Lastly, yes, I created a Workspace and proceeded to invite my colleague, so I should be the owner of it. I first tried inviting him as an owner, then as a viewer. None of which worked.
